app/src/main/res/layout/lr_phase_shift_dialog.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
RC phase shift dialog and LR Phase shift dialog are similar right?
Please inspect yourself if layout are same, reuse it and add a comment that reusing this layout or something to let others know.
Thanks